database/seed.js

```diff
@@ -138,8 +150,6 @@ const seedDb = async () => {
         number_guests: 2,
         cleaning_fee: 15,
         weeknight_discount: .2,
-        addons_names: ['firewood'],
-        addons_price: 15,
         subTotal: 426
       };
       await Booking.create(newBooking)
```

Comment:
i dont know if this newBooking has any information in it. the information from above is not getting stored in the newBooking object created on line 134

Reply:
fixed